pyproject.toml

```diff
@@ -18,6 +18,8 @@ dependencies = [
     "urllib3>=2.4.0",
     "bs4>=0.0.2",
     "requests-kerberos>=0.15.0",
+    "browser-cookie3>=0.19.1",
```

Comment:
the `pdm.lock` file needs to be updated by running `pdm lock`.

Reply:
done